### PR TITLE
minor formatting / writing edits

### PR DESCRIPTION
**Pull request by thedriftofwords**
Cleaning up the commit history on this was... an adventure. Hopefully it looks okay on your end :)

### CODE REVIEW

File: site/en/r2/tutorials/distribute/multi_worker.ipynb [hunk 1]
```diff
@@ -268,69 +268,74 @@
...
     {
+      "cell_type": "markdown",
       "metadata": {
         "colab_type": "text",
         "id": "UhNtHfuxCGVy"
       },
-      "cell_type": "markdown",
       "source": [
-        "## MultiWorkerMirroredStrategy\n",
+        "## `MultiWorkerMirroredStrategy`\n",
```
Comment (lamberta) [inline site/en/r2/tutorials/distribute/multi_worker.ipynb]: No code in headers. There are formatting issues across github/colab/site

File: site/en/r2/tutorials/distribute/multi_worker.ipynb [hunk 2]
```diff
@@ -346,37 +351,39 @@
...
-        "*   The batch size specified here is per-GPU.  In general, the largest batch size that fits the GPU memory is advisable.\n",
-        "*    Cast the variables to `tf.float` if possible.  The official ResNet model includes [an example](https://github.com/tensorflow/models/blob/8367cf6dabe11adf7628541706b660821f397dce/official/resnet/resnet_model.py#L466) of how this can be done.\n",
-        "\n",
-        "\n",
-        "\n"
+        "*   **Increase the batch size:** The batch size specified here is per-GPU.  In general, the largest batch size that fits the GPU memory is advisable.\n",
```
Comment (lamberta) [inline site/en/r2/tutorials/distribute/multi_worker.ipynb]: Remove the bold styles here in the list (italics/emphasis is fine). It visually competes with the headings on the site design

File: site/en/r2/tutorials/distribute/multi_worker.ipynb [hunk 3]
```diff
@@ -175,31 +175,31 @@
...
         "\n",
-        "For illustration purposes, this tutorial shows how one may set a `TF_CONFIG` with a single worker on `localhost`.  In practice, users would create multiple workers on an external IP address and port, and set `TF_CONFIG` on each worker appropriately, i.e. modify the task `index`.\n",
+        "For illustration purposes, this tutorial shows how to set a `TF_CONFIG` with a single worker on `localhost`.  In practice, you would create multiple workers on an external IP address and port, and set `TF_CONFIG` on each worker appropriately, i.e. modify the task `index`.\n",
         "\n",
-        "Warning: Do not execute the following code in Colab.  TensorFlow's runtime will attempt to create a gRPC server at the specified IP address and port, which will likely fail.\n",
+        "**Warning:** *Do not execute the following code in Colab.*  TensorFlow's runtime will attempt to create a gRPC server at the specified IP address and port, which will likely fail.\n",
```
Comment (lamberta) [inline site/en/r2/tutorials/distribute/multi_worker.ipynb]: "Warning: "
(devsite style callout)

File: site/en/r2/tutorials/distribute/multi_worker.ipynb (hunk 1 above)
Comment (thedriftofwords) [reply site/en/r2/tutorials/distribute/multi_worker.ipynb]: Ah, good to know.

File: site/en/r2/tutorials/distribute/multi_worker.ipynb (hunk 2 above)
Comment (thedriftofwords) [reply site/en/r2/tutorials/distribute/multi_worker.ipynb]: done

File: site/en/r2/tutorials/distribute/multi_worker.ipynb (hunk 3 above)
Comment (thedriftofwords) [reply site/en/r2/tutorials/distribute/multi_worker.ipynb]: done

**Comment by lamberta** [comment]
Whoops, sorry.
Can you also apply your changes to the alpha doc for training_loops.ipynb ? https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/distribute/training_loops.ipynb
Thanks

**Comment by lamberta** [comment]
Merge conflict :/

**Pull request merged**